ui/src/app/app-routing.module.ts

```diff
@@ -142,6 +142,9 @@ const routes: Routes = [
                       // use the (urldecoded) captured value nbName
                       pathElementForTitle: 'nbName',
                       breadcrumb: BreadcrumbType.Notebook,
+                      helpContent: 'notebookStorage',
+                      notebookHelpSidebarStyles: true,
+                      notebookRouterOutletContainerStyles: true,
```

Comment:
Is there a more semantic name that can be used here, e.g. "notebookFullscreenStyle" (or something similar - I don't follow the goal for the property). I'm not following how this relates to a router outlet.

Reply:
How about something like `fullHeightOverride`?

Reply:
OK I see what it's doing now. Maybe `containerFullHeightOverride` or `contentFullHeightOverride`? 

Comment in workspace-wrapper wouldn't hurt as well to explain that it's needed primarily due to the notebooks page iframe